_includes/nav.html

```diff
@@ -1,8 +1,8 @@
 {%- if site.texture.showNav -%}
 <div class="navbar">
 	<ul>
-		<a href="#"><li>Home</li></a>
-		<a href="#"><li>About</li></a>
+		<a href="#1"><li>Home</li></a>
```

Comment:
Sorry, But I don't understand the purpose of these changes. Can u please explain?

Reply:
Sure. It would be helpful in maintaining the hashtag number.